services/user.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = "",
+        last_name: str = "",
+) -> None:
+
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name,
+    )
+
+
+def get_user(user_id: int) -> User:
+
+    return get_user_model().objects.get(pk=user_id)
```

Comment:
The 'get_user' function should handle the case where a user with the given ID does not exist. Consider using 'get_object_or_404' or handling the 'DoesNotExist' exception to provide a more user-friendly error message.